src/test/java/org/magicdgs/readtools/tools/quality/QualityEncodingDetectorIntegrationTest.java

```diff
@@ -58,22 +58,22 @@ public void testFileDoesNotExists(final String fileName) throws Exception {
         runCommandLine(Arrays.asList("-I", fileName));
     }
 
-    // TODO: generate a new data provider for this tool
-    // TODO: this is a very simple test for check that we are not changing anything
-    // TODO: but it is not required anymore
-    @DataProvider(name = "smallFiles")
+    @DataProvider(name = "filesWithQualities")
```

Comment:
For this data provider, we will need a Solexa formatted file. We should open a ticket for testing that format!